python/cugraph/components/cc_wrapper.pyx

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c_cc cimport *
+#from libcpp cimport bool
+from c_graph cimport *
+#from libcpp cimport cugraph_cc_t
+from libc.stdint cimport uintptr_t
+import cudf
+#from pygdf import Column
+import numpy as np
+
+cpdef weak_cc(G, connect_type = CUGRAPH_WEAK):
+    """
+    Find the distances and predecessors for a breadth first traversal of a graph.
```

Comment:
seems like the description needs to be updated

Reply:
Ok, I'll fix that, but I don't understand the comment "gpuCI/cugraph/check-style need to be addressed". What is that?

Reply:
There is a python style check (like clang for python) in rapids projects. You can take a look at the failing log and see what should be fixed. It is usually about spacing and indentation stuff.

Reply:
In cugraph/python/cugraph

Try

find . -name "*.py"|xargs flake8

(assuming that flake8 is installed on your system). flake8 checks whether the python files comply to the PEP8 style.

Reply:
Ok, fixed the styling.